Tasks/JavaToolInstallerV0/task.json

```diff
@@ -114,7 +114,7 @@
             "defaultValue": "",
             "required": true,
             "visibleRule": "jdkSourceOption != PreInstalled",
-            "helpMarkDown": "Destination directory into which JDK should be extracted."
+            "helpMarkDown": "On Linux and Windows, it is used as destination directory for JDK installation. On macOS, this directory is used as a temporary folder for extracting .dmg's."
```

Comment:
Maybe something like this would read better:
On Linux and Windows, this is used as the destination directory for JDK installation. On macOS, since there is no way to set the directory for installing the JDK, this directory is used as a temp folder for extracting .dmg's.

Reply:
I've changed your variant a little bit.

Reply:
Fixed grammar a little bit:
On Linux and Windows, this is used as a destination directory for JDK installation. On macOS, this directory is used as a temporary folder for extracting .dmg's since macOS **doesn't** support JDK installation directory changing.

Reply:
Thank you! Fixed